variants/frontend-base/template.rb

```diff
@@ -149,6 +148,28 @@
 
 File.write("./package.json", JSON.generate(package_json))
 
+# fix js lint issues with generated defaults before linting runs
+gsub_file "babel.config.js",
+          "module.exports = function(api) {",
+          "module.exports = api => {"
+
+gsub_file "app/frontend/channels/index.js",
+          "/_channel\\.js$/",
+          "/_channel\\.js$/u"
+
+prepend_to_file "postcss.config.js" do
+  <<~ESLINTFIX
+  /* eslint-disable global-require */ 
+  'use strict'; \n
+  ESLINTFIX
+end
+
+prepend_to_file "babel.config.js", "'use strict';"
+prepend_to_file "config/webpack/development.js", "'use strict';"
+prepend_to_file "config/webpack/environment.js", "'use strict';"
+prepend_to_file "config/webpack/production.js", "'use strict';"
+prepend_to_file "config/webpack/test.js", "'use strict';"
```

Comment:
Just checking that `run "npm run js-lint-fix"` wouldn't be able to fix all these?

Reply:
unfortunately not, that gets run just after these lines but didnt fix these particular issues

Reply:
Adding `'use strict'` is a potentially breaking change, so doesn't have an autofix :)

Reply:
:thumbsup:

Reply:
Do these a newline after them?

Reply:
No, as this'll be handled by `prettier`.